### ISSUE TITLE
spindle pole body separation

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#13461]
GO:0000073
Name	spindle pole body separation

is defined
A largely uncharacterized process involving the release of duplicated spindle pole bodies (SPBs) and their migration away from each other within the nuclear membrane. Duplicated SPBs are connected by a bridge structure that may be severed in order to release the SPBs from one another. Following liberation, SPBs diffuse through the nuclear membrane until they are across from each other. SPB separation must take place in order for a **bipolar mitotic spindle** to assemble

(i.e it is defined  as the mitotic process)

Could this be renamed to
Name	mitotic spindle pole body separation

and positioned under
GO:1905047
Name	mitotic spindle pole body organization

**Comment by ukemi** [comment]
It is not defined as a mitotic process. It just says that it is necessary for a mitotic spindle to assemble. Does it also happen in meiotic divisions?

**Comment by ValWood** [comment]
It does so maybe we need a new term and keep this one agnostic.
We can move existing annotation down...

**Comment by vanaukenk** [comment]
Hi @ValWood 

Would the following edits work:

GO:0000073 spindle pole body separation (remove the mitotic bit from the def)
--GO:new mitotic spindle pole body separation (include mitotic bit)
--GO:new meiotic spindle pole body separation (include meiotic bit)

and make sure that each of the new terms has the proper cell cycle parent.  

I noticed, too, that there are annotations to regulation terms for 'spindle pole body separation' that could probably also be moved to more specific 'mitotic' terms, if those were created, as well.

Thoughts?

**Comment by ValWood** [comment]
Yes this would be great.

 We could potentially flag ALL of these non meiotic/mitotic parent terms as "not for direct annotation" and many more "grouping" terms in the cell cycle branch. I think this would be a good plan as part of the guidelines for annotation which come out of the cell cycle work.
 A couple more fixes will be required in the checkpoint branches....will open a ticket for this once I get back from pombe 2017 (and while here will discuss with a few more checkpoint people).

**Comment by ValWood** [comment]
Hi Kimberley,  
could these 2 children be added? I want to migrate to the mitotic term before we get to many.
Cheers
Val

**Comment by vanaukenk** [comment]
@ValWood   Sure, I can add these this week.

Aside from updating wrt the statement "SPB separation must take place in order for a bipolar mitotic spindle to assemble.", is there anything else we should update in the definitions?  Is it still a largely uncharacterized process?

Here's what I am planning to add, along with the respective regulation terms.  For the latter, there are only annotations from SGD and UniProt that could potentially be housed to a more specific term.  I will add an issue in go-annotation for that.

GO:0000073 spindle pole body separation
-GO:new mitotic spindle pole body separation
-GO:new meiotic spindle pole body separation

GO:1903047 mitotic cell cycle process
-GO:new mitotic spindle pole body separation

GO:1903046 meiotic cell cycle process
-GO:new meiotic spindle pole body separation

**Comment by ValWood** [comment]
We don't see to know that much about the mechanism of the  process AFAIK , but we can say that it's "microtubule -dependent" in the def as 6/7 genes with the phenotype abnormal/abolished spindle pole body separation are microtubule organising thingys.

**Comment by vanaukenk** [comment]
Okay, how about this for the 'base' definition:

A microtubule-dependent process, largely uncharacterized, involving the release of duplicated spindle pole bodies (SPBs) and their migration away from each other within the nuclear membrane. Duplicated SPBs are connected by a bridge structure that may be severed in order to release the SPBs from one another. Following liberation, SPBs diffuse through the nuclear membrane until they are across from each other.

**Comment by ValWood** [comment]
I'm going to check this out with Takashi Toda....I reilise I know less about this than I thought!

**Comment by vanaukenk** [comment]
Okay, just let me know if/when you need any additional changes to the defs. or placement.

**Comment by ValWood** [comment]
Yep the def sounds a bit odd. It doesn't seem likely that SPBs diffuse through the nuclear envelope, they are  localized because their movement is "directed" (thats the MT-dependent part). That would actually make this a type of microtuble-dependent localization.

**Comment by ValWood** [comment]
deleted previus comment, din't mean to post,!...and this one...

**Comment by ValWood** [comment]
So, revised definition
I think we should rename this to
“initial mitotic spindle pole body separation” (& co)
because "spindle pole body separation could be taken to mean the whole of  the phase 1 spindle elongation.


Then  based on Takashi's defs

1. NTR: insertion of  duplicated spindle pole body into nuclear envelope
DEF insertion of the spindle pole body from the cytoplasmic site upon mitotic entry 
TAXON RESTRICTION: never in saccharomyces occurs in pombe, cerevisiae is different

2. Initial mitotic spindle pole body separation
DEF 
begins with the nucleation of microtubules  from each SPB within the nucleus, leading to V-shaped spindle microtubules. Interpolar microtubules that elongate from each pole are interconnected, forming overlapping microtubules. Capturing and antiparallel sliding apart of microtubules promotes the initial separation of the SPB. 
(I think meiosis will be similar but we could wait to create that term)

 3.
 This "initial separation" is really part of the  slow pre-anaphase spindle elongation represented by:
GO:0061804 mitotic spindle elongation during mitotic prophase
is defined
The cell cycle process in which the distance is lengthened between poles of the mitotic spindle during mitotic prophase. 
(it must be part of this process, because the distance between the spindle poles increases)

4. Takashi also suggests that we revise GO:0061804 
from
mitotic spindle elongation during mitotic prophase
to
mitotic spindle elongation during mitotic prophase and prometaphase

5. I think we only need the word "mitotic" in the elongation terms once

**Comment by ValWood** [comment]
This is the temporal order of events for a closed mitosis

1: insertion of  duplicated spindle pole body into nuclear envelope (in some species)

2:  mitotic spindle elongation during mitotic prophase and prometaphase
 --- 2: Initial mitotic spindle pole body separation

3: Chromosome segregation part (lots of terms refer to  this part, I will order them later once I finish curating mitotic chromosome segregation papers...200 and counting...)

4. mitotic spindle elongation during mitotic anaphase (when the distance is lengthened beyond the existing nuclear envelope)

**Comment by ValWood** [comment]
becasue it is the same term I'll  add 
https://github.com/geneontology/go-ontology/issues/14086
seems odd for different people to be editing the same term differently


- [ ] revise definition for mitotic spindle assembly
Current Def
The aggregation, arrangement and bonding together of a set of components to form the spindle that contributes to the process of mitosis.

Revised Def
Mitotic bipolar spindle assembly begins with spindle microtubule nucleation from the separated spindle pole body, includes spindle elongation during prometaphase, and is complete when all kinetochores are stably attached the spindle, and the spindle assembly checkpoint is satisfied.

 - [ ] This means that
i) GO:0061804 mitotic spindle elongation during mitotic prophase
should be part of spindle assembly.

However Takashi also says that the "spindle midzone assembly" term might be problematic because it is possible that this occurs after bipolarity is achieved, at which point the spindle is considered to be assembled.

- [ ] It might seem odd for "mitotic spindle midzone assembly" not to be descendant of "mitotic spindle assembly"
Perhaps the primary term name could be should renamed to "mitotic spindle polar microtubule interdigitation"?
and remove it as a descendent of "mitotic spindle assembly"

**Comment by ValWood** [comment]
Sorry Kimberley, not so quick now!  Also I thing SGD Rob Nash should sanity check this for S. cerevisiae.

**Comment by ValWood** [comment]
@srengel (I can't see Rob's ID here https://github.com/geneontology/go-site/blob/master/metadata/group-contacts.csv)

Could you ask Rob to check that this would make sense for cerevisiae?

cheers

Val

**Comment by robnash** [comment]
Hi,

There is a lot in this item and I am not sure what specifically I have been asked to comment on. There is really nice review forced on the SPB cycle in both budding and fission yeast. 

Cavanaugh AM and Jaspersen SL (2017) Big Lessons from Little Yeast: Budding and Fission Yeast Centrosome Structure, Duplication, and Function. Annu Rev Genet 51: 361-383.
PMID: 28934593

The senior author, Sue Jaspersen (slj@stowers.org), is an expert in the field and is very knowledgeable. If there are specific questions I could contact her to see if she could help clarify. I would assume Takashi Toda is also very familiar with the SPB cycle in _S. cerevisiae_? 

I am fine with idea of separating the mitotic and meiotic terms, as this is common practice in other areas of GO, although I am not sure how different the mitotic and meiotic processes really are.

Minor point but I don’t like the phrase “largely uncharacterized” in the definition. This area has moved quite rapidly and much more is known now so I would prefer to leave this bit out. 

I disagree with the need for the word “initial” in the term “initial mitotic spindle pole body separation”. I don’t think users would confuse this will spindle elongation. They are distinct. I don’t think researchers refer to this is initial. Is it possible to add a bit in the definition like “Not to be confused with phase 1 spindle elongation.” or something along these lines?

In the comments on Sep 6, based on Takashi’s feedback, why is this purposed NTR specified as taxon restricted:

NTR: insertion of duplicated spindle pole body into nuclear envelope
DEF insertion of the spindle pole body from the cytoplasmic site upon mitotic entry
TAXON RESTRICTION: never in saccharomyces occurs in pombe, cerevisiae is different
  
Since this occurs in both pombe and cerevisiae it would be a shame if we would not use it for both yeasts. From this review:

Jaspersen SL and Ghosh S (2012) Nuclear envelope insertion of spindle pole bodies and nuclear pore complexes. Nucleus 3(3):226-36 PMID: 22572959

I get the impression that in pombe the SPB is tethered to the nuclear envelope (NE) throughout the cell cycle but only instead into the NE prior to entry into mitosis. So, there does appear to be a difference in timing, as budding yeast SPB duplication and insertion occur in late G1 phase. Is there more to it than this? If its just temporal couldn’t the definition be written without the temporal aspect? 

Cheers,
Rob

**Comment by ValWood** [comment]
> There is a lot in this item and I am not sure what specifically I have been asked to comment on. 

All of it.

Thanks, these comments are really useful. I'll revise based on your input.

I'll also run the revised version by the Jaspersen lab since I am in contact with them anyway- (they also work on pombe now, in fact we are currently featuring their "SPB/centrosome assembly model on the PomBase front page
https://www.pombase.org/ or  https://www.pombase.org/ if it missed the reload cycle...its pretty cool stuff )

**Comment by robnash** [comment]
Thanks Val.  I like these home page highlights. Sue was a grad student in the lab where I did my post-doc and I got to see here more recently in the summers while she was an instructor at the Yeast Genetics & Genomics Course. Really sharp scientist with lots of knowledge ticked away.

**Comment by ValWood** [comment]
Hi @vanaukenk ,

I  edited the above in accordance with Robs comments. I think we are good to go now. I removed anything inconsistent.

It would be great to get these as I'm doing loads of papers in this area and  now I have a  large set to resolve/add these terms. Let me know if you need me to clarify or Skype  when you add.
v

**Comment by vanaukenk** [comment]
@ValWood 
Okay, I will resume work on this ticket and let you know if I have any questions.

**Comment by ValWood** [comment]
OK I should be able to answer questions on this straight away as I have done a few spindle elongation papers this week.... these terms will be great to sort, it's a bit confusing  now;)

**Comment by ValWood** [comment]
one more thing for this, which came to mind  today.


GO:0061805 | mitotic spindle elongation during mitotic anaphase
and
GO:0061804 mitotic spindle elongation during mitotic prophase

term names sound a bit weird. It makes them sound like the same process occuring at different times, but this it the 2 separate stages of spindle elongation (which are mechanistically different from each other).

initial spindle formation  (phase one or prophase), 
a period of constant spindle length (phase two or metaphase plus anaphase A), 
and spindle elongation (phase three or anaphase B)

including phase one and phase three  in term names might also seem a bit odd (these could be synonyms?)

instead perhaps they would be better named as
mitotic prophase spindle elongation
and
mitotic anaphase spindle elongation

**Comment by ValWood** [comment]
most papers call the first stage of elongation "formation" so I x-d out the final part.

For example the paper I am curating now is

PMID:27697865
"To determine if spindle formation (spindle phase I) and spindle elongation in anaphase B (phase III) were modulated by Asp1 kinase function..."

Hi @vanaukenk  is there anything you need from me here before this is implemented?

**Comment by ValWood** [comment]
I would like on final  suggested change reinstated

previously, Takashi  Toda suggested that we rename 
"mitotic spindle pole body separation" to 
"Initial mitotic spindle pole body separation"

@robnash  said:

>I disagree with the need for the word “initial” in the term “initial mitotic spindle pole body separation”. I don’t think users would confuse this will spindle elongation. They are distinct. I don’t think researchers refer to this is initial. Is it possible to add a bit in the definition like “Not to be confused with phase 1 spindle elongation.” or something along these lines?

However,
The paper I am reading now  used  the phrase "spindle pole body separation" to desctibe phase I (I'm sure I have seen this a number of times already):

![4b](https://user-images.githubusercontent.com/7359272/38251290-f708c210-3748-11e8-8040-94a82b3a7302.jpg)

" However, SPB separation in asp1D333A cells was significantly faster; the average speed of separation was 0.50 􏰐 0.06 􏰊m/min (Fig. 4B). Spindle phase I was not changed significantly in asp1H397A cells compared to wild-type asp1􏰋 cells (Fig. 4B) (average speed of SPB separation in asp1H397A cells, 0.43 􏰐 0.06 􏰊m/min)."


So, for complete unambiguity I think we should call this term "initial spindle pole body separation"

**Comment by ValWood** [comment]
for @vanaukenk , the GO term refers only to this part.
https://github.com/geneontology/go-ontology/issues/15304

**Comment by robnash** [comment]
When the dust has settled on this reorganization could someone provide a final summary of terms added, and their relationshops in the hierarchy? At some point we will probably need to consider reannotation and it would be nice to have this in a summarized form instead of having this in multiple Github postings and have to jump from item to item to piece it all together. Thanks.

**Comment by ValWood** [comment]
Hi Rob/Kimberley

This is the summary.Changest should not affect existing annotations, this is only to
make the terms clearer and fix incorrect parentage.

 - [ ] 1. *initial * mitotic spindle pole body separation
DEF update
begins with the nucleation of microtubules  from each SPB within the nucleus, leading to V-shaped spindle microtubules. Interpolar microtubules that elongate from each pole are interconnected, forming overlapping microtubules. Capturing and antiparallel sliding apart of microtubules promotes the initial separation of the SPB. 
(I think meiosis will be similar but we could wait to create that term)

 - [ ] 2.
 This initial "mitotic spindle pole body separation" is really part_of the  slow pre-anaphase spindle elongation represented by:
GO:0061804 mitotic spindle elongation during mitotic prophase
is defined
The cell cycle process in which the distance is lengthened between poles of the mitotic spindle during mitotic prophase. 
(it must be part of this process, because the distance between the spindle poles increases)

so, make GO:0061804 mitotic spindle elongation during mitotic prophase
a parent of "mitotic spindle pole body separation" 


 - [ ] 3. Takashi also suggests that we revise GO:0061804  name
from
mitotic spindle elongation during mitotic prophase
to
mitotic spindle elongation during prophase and prometaphase

we could name this 
mitotic spindle formation (spindle phase one)
synonym mitotic spindle elongation during prophase
synonym mitotic spindle elongation during prophase and prometaphase
(We only need the word "mitotic" in the elongation terms/synonyms once)

and
from
GO:0061805 | mitotic spindle elongation during mitotic anaphase
to
mitotic spindle elongation (spindle phase three)
synonym mitotic anaphase spindle elongation
synonym mitotic spindle elongation during mitotic anaphase

- [ ] 4.revise definition for mitotic spindle assembly
Current Def
The aggregation, arrangement and bonding together of a set of components to form the spindle that contributes to the process of mitosis.

Revised Def
Mitotic bipolar spindle assembly begins with spindle microtubule nucleation from the separated spindle pole body, includes spindle elongation during prometaphase, and is complete when all kinetochores are stably attached the spindle, and the spindle assembly checkpoint is satisfied.

 - [ ] 5. This means that
i) GO:0061804 mitotic spindle elongation during mitotic prophase
should be part of mitotic spindle assembly.

- [ ]  6.  However Takashi also says that the "spindle midzone assembly" term might be problematic because it is possible that this occurs after bipolarity is achieved, at which point the spindle is considered to be assembled.
It might seem odd for "mitotic spindle midzone assembly" not to be descendant of "mitotic spindle assembly" ?
Perhaps the primary term name could be should renamed to "mitotic spindle polar microtubule interdigitation"?
and remove it as a descendent of "mitotic spindle assembly"

**Comment by robnash** [comment]
Thanks Val

**Comment by ValWood** [comment]
7.
also now need regulation terms for
GO:0061804 mitotic spindle elongation during mitotic prophase (but will be renamed above)
and
GO:0061805 | mitotic spindle elongation during mitotic anaphase
for PMID:27697865

**Comment by ValWood** [comment]
Hi @vanaukenk   do you need anything else for this one? It would be good to get the revisions as T. Toda made these suggestions a long while ago and it would improve some of the representation and use of these terms.

**Comment by ValWood** [comment]
Hi @vanaukenk  this one is prettly fleshed out (refer to the final summary).
Can we implement, I would like to resume my chromosome segregation work soon, but I wianted to make sure the ontology fixes were in place first.
Cheers
Val

**Comment by ValWood** [comment]
Hi Kimberley.

Can we implement these changes? They  should be quite straightforward if you begin from my summary comment:

ValWood commented on Apr 3, 2018
Hi Rob/KimberleyThis is the summary.
--

and the one additional action (point7 below)

Each item should be fairly self-contained, but probably good to do in the order as there might be some dependencies.

It would be  good to make these refinements as Takashi Toda spent quite a bit of time revising defs etc for us and this was 2 years ago. Takashi is a  world expert in mitotic spindle and understands GO quite well, he does lots of community curation.

If you need any further  input I can easily comment, but I am removing myself as assignee as there isn't a task for me right now.

**Comment by ValWood** [comment]
OK re point 6, let's leave this for now. If it becomes confusing we can address later, but these changes will be helpful.

**Comment by ValWood** [comment]
@vanaukenk  I could have sworn you did these changes recently. Was it another ticket?

**Comment by vanaukenk** [comment]
@ValWood 
I thought so, too.
I'll take another look but probably not until after the meeting.

**Comment by ValWood** [comment]
I noticed this week that this now seems done. I still think point 6 might be a problem, but I can open a new ticket for that one.

Thanks @vanaukenk !
